bin/upgrading/update-db.pl

```diff
@@ -701,47 +701,6 @@ sub populate_moods {
     }
 }
 
-sub populate_mogile_conf {
-    # create/update the MogileFS database if we use it
-    return unless defined $LJ::MOGILEFS_CONFIG{hosts};
```

Comment:
MogileFS is deprecated. People need to migrate to plain-store or S3 (to be included soon). I'm not supporting MogileFS going forward. So, this code doesn't have any purpose -- it's only useful for setting up a new MogileFS cluster.